src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.html

```diff
@@ -132,7 +132,26 @@
                                 [messages]="messages"></cd-select-badges>
             </span>
           </div>
+        </div>
 
+        <!-- Enabled -->
```

Comment:
Please use a common checkbox here, a radio group button does not display more information to the user here.

Reply:
It definitely makes sense to have that unified and if we currently use checkboxes in Ceph Dashboard instead of radio group buttons, I should have implemented checkboxes. 

But because I personally prefer to have radio buttons instead of standard checkboxes, I'd like to ask you if we could reconsider using them instead of checkboxes for simple on/off toggles.

Material design has a [page](https://material.io/design/components/selection-controls.html#) dedicated to this topic. On that page it is explained when to use checkboxes, radio buttons and switches (I consider **two** radio buttons in a group of the Bootstrap framework to be a switch). The document says:

> Use switches to:
>
> Toggle a single option on or off, on mobile and tablet
> Immediately activate or deactivate something

About radio buttons they say:

> Radio buttons should be used instead of checkboxes if only one item can be selected from a list.

This leaves some room for interpretation and I know that we do not follow the material design guide, but as I personally think they are doing a great job and that it is a great resource, we could at least give it a thought.

Let me know what you think and please excuse me starting that discussion in this PR.

Reply:
> Immediately activate or deactivate something

If we follow this guideline, I would only use a switch on checkboxes like the ones in table column filtering. Since those are immediately changing the table.

In the case of forms, we are not immediately changing anything, the change only occur when you click the submit button.